### DISCUSSION TITLE
Show progress feature

### DISCUSSION THREAD

**Discussion by nonoumasy** [benyap/firestore-backfire#346] [Ideas]
Just a suggestion, it would be nice to show a progress of the export/import.
something simple like showing which document has been processed in a new line would be good enough

**Comment by benyap** [comment]
Hi @nonoumasy, have you tried using the [`--debug` or `--verbose` option](https://github.com/benyap/firestore-backfire#logging-options)?

This should print regular updates on the number of documents that have been analysed and exported as it's running.

It wasn't practical to do a console.log for every document that was processed, and it often also slowed down the process - outputting to the console is more expensive than you'd think 🥲. So the current implementation was the best solution I've found in my experience. Hopefully this is sufficient!

![image](https://github.com/benyap/firestore-backfire/assets/19235373/17b09fcc-bb04-419b-a989-1e6c706c055e)
